typescript/web/src/connectors/undo-store/effects/label-and-label-class.fixtures.ts

```diff
@@ -0,0 +1,140 @@
+import { FetchResult } from "@apollo/client";
+import { v4 as uuid } from "uuid";
+import { MATCH_ANY_PARAMETERS } from "wildcard-mock-link";
+import {
+  CreateLabelClassActionMutation,
+  CreateLabelClassActionMutationVariables,
+} from "../../../graphql-types/CreateLabelClassActionMutation";
+import {
+  DeleteLabelClassActionMutation,
+  DeleteLabelClassActionMutationVariables,
+} from "../../../graphql-types/DeleteLabelClassActionMutation";
+import {
+  GetLabelIdAndClassIdQuery,
+  GetLabelIdAndClassIdQueryVariables,
+} from "../../../graphql-types/GetLabelIdAndClassIdQuery";
+import {
+  UpdateLabelClassActionMutation,
+  UpdateLabelClassActionMutationVariables,
+} from "../../../graphql-types/UpdateLabelClassActionMutation";
+import {
+  ApolloMockResponse,
+  ApolloMockResponses,
+} from "../../../utils/tests/apollo-mock";
+import {
+  BASIC_LABEL_DATA,
+  DEEP_DATASET_WITH_CLASSES_DATA,
+} from "../../../utils/fixtures";
+import {
+  CREATE_LABEL_CLASS_QUERY,
+  DELETE_LABEL_CLASS_MUTATION,
+  GET_LABEL_QUERY,
+  UPDATE_LABEL_MUTATION as UPDATE_LABEL_MUTATION_SHARED,
+} from "./shared-queries";
+import { UPDATE_LABEL_MUTATION as UPDATE_LABEL_CLASS_OF_LABEL_MUTATION } from "./update-label-class-of-label";
+
+const [PREVIOUS_LABEL_CLASS_DATA] = DEEP_DATASET_WITH_CLASSES_DATA.labelClasses;
```

Comment:
Can't we avoid this by extracting the inner label-classes of  `DEEP_DATASET_WITH_CLASSES_DATA` in `*_LABEL_CLASS_DATA` constants?

(_please update your branch with the latest revision of #843 if you decide to do it 🙏_)

Reply:
I agree with that and as I said in a comment from my previous PR this should be done as a separate task, since it involves quite a lot of changes to all the tests and fixtures files.

I _could_ do it in this PR though, since it's one of the last in the drop jest tests collection.